great_expectations/rule_based_profiler/util.py

```diff
@@ -220,6 +220,44 @@ def compute_bootstrap_quantiles(
     false_positive_rate: np.float64,
     n_resamples: int,
 ) -> tuple:
+    """
+    Internal implementation of the "bootstrap" estimator method, returning confidence interval for a distribution.
+    See https://en.wikipedia.org/wiki/Bootstrapping_(statistics) for an introduction to "bootstrapping" in statistics.
+
+    This implementation is sub-par compared to the one available from the "SciPy" standard library
+    ("https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.bootstrap.html"), because it introduces bias and
+    does not handle multi-dimensional statistics (unlike "scipy.stats.bootstrap", which corrects for bias and is
+    vectorized, thus having the ability to accept a multi-dimensional statistic function and process all dimensions).
+
+    This implementation will be replaced by "scipy.stats.bootstrap" when Great Expectations can be upgraded to use a
+    more up-to-date version of the "scipy" Python package (the currently used version does not have "bootstrap").
+
+    Additional future direction (potentially as a contribution submission to the "SciPy" community) include developing
+    enhancements to bootstrapped estimator based on theory presented in "http://dido.econ.yale.edu/~dwka/pub/p1001.pdf":
+    @article{Andrews2000a,
+        added-at = {2008-04-25T10:38:44.000+0200},
+        author = {Andrews, Donald W. K. and Buchinsky, Moshe},
+        biburl = {https://www.bibsonomy.org/bibtex/28e2f0a58cdb95e39659921f989a17bdd/smicha},
+        day = 01,
+        interhash = {778746398daa9ba63bdd95391f1efd37},
+        intrahash = {8e2f0a58cdb95e39659921f989a17bdd},
+        journal = {Econometrica},
+        keywords = {imported},
+        month = Jan,
+        note = {doi: 10.1111/1468-0262.00092},
+        number = 1,
+        pages = {23--51},
+        timestamp = {2008-04-25T10:38:52.000+0200},
+        title = {A Three-step Method for Choosing the Number of Bootstrap Repetitions},
+        url = {http://www.blackwell-synergy.com/doi/abs/10.1111/1468-0262.00092},
+        volume = 68,
+        year = 2000
+    }
+    The article outlines a three-step minimax procedure that relies on the Central Limit Theorem (C.L.T.) along with the
+    bootsrap sampling technique (please see https://en.wikipedia.org/wiki/Bootstrapping_(statistics) for background) for
```

Comment:
Typo in "bootstrap"

Reply:
@NathanFarmer Nice catch!!!  Fixed!  Thank you!